src/main/java/com/carservice/manager/config/Log4J2YamlConfig.java

```diff
@@ -0,0 +1,33 @@
+package com.carservice.manager.config;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Log4J2YamlConfig {
+
+    private Logger logger = LogManager.getLogger();
+
+    public void performSomeTask() {
+        logger.debug("This is a debug message");
+        logger.info("This is an info message");
+        logger.warn("This is a warn message");
+        logger.error("This is an error message");
+        logger.fatal("This is a fatal message");
+    }
+
+    public void infoLogEnterIntoMethod(String methodName) {
+        logger.info("Enter into " + methodName + " method.");
```

Comment:
With the 'info' level we can pollute logs a bit 😉 It is worth considering lowering the level (to debug or trace).
And the same with infoLogSuccess.